src/components/widgets/NumericInput.js

```diff
@@ -12,13 +12,25 @@ export default class NumericInput extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {value: props.value};
+    this.state = {
+      value: props.value,
```

Comment:
I know this is not a new code addition, but assigning props to state is such an antipattern.

Reply:
that's our way of not having plotly.js reject the value and override with its default